docs/rules/naming/var-name-mixedcase.md

```diff
@@ -12,7 +12,7 @@ title:       "var-name-mixedcase | Solhint"
 
 
 ## Description
-Variable name must be in mixedCase. (Does not check IMMUTABLES, use immutable-vars-naming)
+Variable names must be in mixedCase. (Does not check IMMUTABLES, use immutable-vars-naming)
 
```

Comment:
`This is a dynamically generated file. Do not edit manually.` 
You can see this legend at the top of the file